dockers/docker-sonic-telemetry/base_image_files/monit_telemetry

```diff
@@ -2,4 +2,4 @@
 ## Monit configuration for telemetry container
 ###############################################################################
 check program container_memory_telemetry with path "/usr/bin/memory_checker telemetry 419430400"
-    if status == 3 for 10 times within 20 cycles then exec "/usr/bin/restart_service telemetry"
+    if status == 3 for 10 times within 20 cycles then exec "/usr/bin/restart_service telemetry" repeat every 2 cycles
```

Comment:
>repeat [](http://example.com/codeflow?start=96&length=6)

You mentioned this is a monit feature. Please make sure monit on all the backport branches have this feature.

Reply:
I will double check whether this new syntax existed on 201811, 201911 and 202012 images or not.

Reply:
In 201811, 201911 and 202012 images, Monit has the same version `5.20.0`. I checked this syntax of Monit with 20191130.83 image on lab device and it did work.